pkg/apis/tensorflow/v1alpha2/defaults.go

```diff
@@ -88,9 +88,19 @@ func setTypeNameToCamelCase(tfJob *TFJob, typ TFReplicaType) {
 
 // SetDefaults_TFJob sets any unspecified values to defaults.
 func SetDefaults_TFJob(tfjob *TFJob) {
+	// Set default cleanpod policy to All.
+	if tfjob.Spec.CleanPodPolicy == nil {
```

Comment:
I think a more sensible default would be to only delete running pods. That seems like the most useful policy because it preserves logs and doesn't consume resources.

Its also consistent with behavior of K8s job.

Reply:
Then we will delete ps and keep workers, make sense to me. But maybe look weird. I will implement it now.

Reply:
Thanks. I agree its weird but the logs of the workers and the chief are probably what you care about anyway and this way users will have logs available by default.